javascript/preflight.js

```diff
@@ -1,7 +1,7 @@
 function runPreFlightChecks() {
   const isHttp      = window.location.protocol == 'http:',
         isLocalhost = window.location.hostname == 'localhost'
-        isUnsafe    = !(isHttp && isLocalhost);
+        isUnsafe    = !isHttp || !(isHttp && isLocalhost);
```

Comment:
I think this condition is wrong (truth tables below). As far as I understand you're aiming for it being marked as unsafe if you are serving over http and not serving over localhost. ie `isUnsafe = isHttp && !isLocalhost;`

Reply:
Bah. You're right. It's not working out `isSafe` is working out `isUnsafe`.

Reply:
https://github.com/alphagov/fourth-wall/pull/61